### DISCUSSION TITLE
Paperless consume stuck in queue

### DISCUSSION THREAD

**Discussion by malosaa** [tteck/Proxmox#1269] [Help Needed] [answered]
Anyone else having this issue ?
As for me since yesterday when uploading the PDF it got stuck in queue with no error at all..
Restarting the full container does solve it but not for long..

I did read somewhere the redis needs to be updated to v7.. when checking here its 6.02XX
Any idea how i can do that ?
As running the default update does say i'm on latest version..

regards

**Comment by tteck** [comment, marked as answer]
In order to upgrade Redis to version 7, it is necessary to use the Debian backport repositories. To do so, you need to add the backport repositories to your system's list of package sources by running the following command in the terminal:

```
cat <<EOF >>/etc/apt/sources.list
deb http://deb.debian.org/debian bullseye-backports main contrib non-free
deb-src http://deb.debian.org/debian bullseye-backports main contrib non-free
EOF
```
After adding the backport repositories, run the `apt update` command to update the package lists. To upgrade to Redis version 7.0.7, run the command `apt -t bullseye-backports install -y redis`. This will install the latest version of Redis from the backport repositories.

**Reply by malosaa** [reply]
Many thanks i did run it and all worked.
Now lets hope the problem is solved.
Many thanks again for the quick reply :) 
👍
